SerpentsHand/EventHandlers.cs

```diff
@@ -100,6 +100,12 @@ internal static void OnTeamRespawn(RespawningTeamEventArgs ev)
                                 SerpentsRespawnCount++;
                         }
                     });
+                    
+                    foreach (var scp in Player.Get(Team.SCP))
+                    {
+                        scp.Broadcast(duration, message);
```

Comment:
You didn't make the message configurable anywhere. There is also no option for the hint as you stated in your decription.

Reply:
Ops... sorry, I will do it right now